guides/common/assembly_administering-project-with-projectwebui.adoc

```diff
@@ -0,0 +1,17 @@
+[id="administering-project-with-projectwebui_{context}"]
+= Getting started with {ProjectWebUI}
+
+Get to know the basics of administering {Project} from the {ProjectWebUI}.
+
+include::modules/proc_importing-the-katello-root-ca-certificate.adoc[leveloffset=+1]
```

Comment:
I'm wondering about this. This is Katello specific, so it should be hidden. We right now hide it like this:
https://github.com/theforeman/foreman-documentation/blob/19b38e6be136208a7b43e8d5bf8c92a8d1f0a682/guides/common/assembly_accessing-server.adoc#L5-L7

But would it make more sense to include those ifdefs in the procedure itself to avoid duplicating it like this:

```suggestion
ifdef::katello,orcharhino,satellite[]
include::modules/proc_importing-the-katello-root-ca-certificate.adoc[leveloffset=+1]
endif::[]
```

Reply:
I created https://github.com/theforeman/foreman-documentation/pull/3090 for this. Let's see if the team agrees.

Reply:
Based on https://github.com/theforeman/foreman-documentation/pull/3090, let's stick to including ifdefs in parent files rather than in the included files themselves.

Reply:
Added.